scraper/main.py

```diff
@@ -5,39 +5,37 @@
 urls = {
     'Facebook': 'https://www.facebook.com/careers/jobs?search_text=internship',
     'Amazon': 'https://www.amazon.jobs/en/search?base_query=internship',
-    'Apple': 'https://jobs.apple.com/en/search?search=internship',
+    'Apple': 'https://jobs.apple.com/en-ca/search?team=internships-STDNT-INTRN',
```

Comment:
Project requirements state that the internship locations should cover US, Canada, and Mexico. 

Can we consider appending search queries to the urls if possible? ie: `&location=united-states-USA+canada-CANC+mexico-MEXC` for apple

Reply:
This would help reduce any unnecessary internships that don't match project scope